### PR TITLE
UT: Add unit test for media service.

### PR DESCRIPTION
**Pull request by xmano**
Added unit test for media service 

Closing the earlier PR #3128

`npx jest media.service.spec.ts`

image

### CODE REVIEW

File: libs/media/src/lib/+state/media.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        })},
+        { provide: SETTINGS, useValue: { host: 'localhost:8080', ssl: false } }
+      ],
+    });
+    db = TestBed.inject(AngularFirestore);
+    storage = TestBed.inject(AngularFireStorage);
+    service = TestBed.inject(MediaService);
+
+    await loadFirestoreRules({
+      projectId: "test",
```
Comment (fritzschoff) [inline libs/media/src/lib/+state/media.service.spec.ts:47]: if possible can you use single quotes? Just out of consistency of this repo.

File: libs/media/src/lib/+state/media.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      rules: readFileSync('./firestore.test.rules', "utf8")
```
Comment (fritzschoff) [inline libs/media/src/lib/+state/media.service.spec.ts:48]: same here if possible

**Review by fritzschoff — APPROVED** [review]
LGTM!

File: libs/media/src/lib/+state/media.service.spec.ts (hunk 2 above)
Comment (xmano) [reply libs/media/src/lib/+state/media.service.spec.ts]: Updated code .. removed loading of rules. So this  & above goes away

File: libs/media/src/lib/+state/media.service.spec.ts (hunk 1 above)
Comment (xmano) [reply libs/media/src/lib/+state/media.service.spec.ts]: code removed.

**Comment by phpgeekfr** [comment]
Yeah good job :)

**Pull request merged**